packages/sui-studio/src/components/layout/_style.scss

```diff
@@ -11,43 +10,49 @@
   &-sidebar {
     @include breakpoint-from(s) {
       flex: 0 0 $w-sidebar;
-      transform: translate3d(0, 0, 0);
+      // transform: translate3d(0, 0, 0);
```

Comment:
Maybe we can remove it if it is no longer used.

Reply:
✅ done